<launch>
    <!-- 1. Add an arg for number of chambers, with a default of 4 -->
    <arg name="ch" default="4" />

    <!-- 2. Launch nodes -->
    <node pkg="pushpull_suctioncup_106a" type="robotStatePublisher.py"
          name="robotStatePublisher" output="screen" />

    <arg name="ATI_ip" default="192.168.1.42" />
    <node pkg="netft_utils" type="netft_node" name="netft_node"
          output="screen" args="$(arg ATI_ip)" />

    <node pkg="pushpull_suctioncup_106a" type="data_logger.py" name="datalogger"
          output="screen" />

    <!-- 3. Use the arg to pick which .xml config to load -->
    <node pkg="rqt_multiplot" type="rqt_multiplot" name="rqt_multiplot_node"
          output="screen"
          args="--multiplot-run-all --multiplot-config $(find pushpull_suctioncup_106a)/launch/rqt_multiplot_ATI_Psensor_ch$(arg ch).xml" />
</launch>
